gazelle_haskell_modules/lang.go

```diff
@@ -186,13 +198,16 @@ func (*gazelleHaskellModulesLang) GenerateRules(args language.GenerateArgs) lang
 		}
 	}
 
-	ruleInfos := rulesToRuleInfos(args.Dir, args.File.Rules, args.Config.RepoName, args.File.Pkg)
+	// The list of rules should contain both the rules already present in the build file (args.File.Rules)
+	// and the one that previous extensions of gazelle generated in this run (args.OtherGen...).
+	// This second argument contains the rule generated by gazelle_cabal when both extensions are ran in the same gazelle_binary.
+	rulesList := append(args.File.Rules, args.OtherGen...)
+	ruleInfos := rulesToRuleInfos(args.Dir, rulesList, args.Config.RepoName, args.File.Pkg)
```

Comment:
Is `gazelle_cabal` guaranteed to run before `gazelle_haskell_modules`? If a specific order is necessary, perhaps we need instructions documented somewhere.